crystal-mode.el

```diff
@@ -2304,144 +2304,149 @@ See `font-lock-syntax-table'.")
     (,(concat
        crystal-font-lock-keyword-beg-re
        (regexp-opt
-        '("alias"
+        '("abstract"
+          "alias"
+          "annotation"
+          "as"
+          "as?"
+          "asm"
           "begin"
           "break"
           "case"
           "class"
-          "context"
           "def"
-          "describe"
-          "defined?"
           "do"
-          "elsif"
           "else"
-          "fail"
+          "elsif"
+          "end"
           "ensure"
+          "enum"
+          "extend"
+          "false"
           "for"
-          "end"
+          "fun"
           "if"
           "in"
-          "it"
+          "include"
+          "instance_sizeof"
+          "is_a?"
           "lib"
           "macro"
           "module"
-          "annotation"
           "next"
+          "nil"
+          "nil?"
           "of"
-          "pending"
-          "redo"
+          "out"
+          "pointerof"
+          "previous_def"
+          "private"
+          "protected"
+          "puts"
+          "require"
           "rescue"
-          "retry"
+          "responds_to?"
```

Comment:
Same thing, `responds_to?` is a method on `Object`.

Reply:
> Same thing, `responds_to?` is a method on `Object`.

Hi, me too, i confusing about why some of them added as keyword, but some of them not.(e.g. loop, spawn)

I just following the latest official document.

https://crystal-lang.org/reference/1.15/crystal_for_rubyists/index.html#available-keywords

The last changes for this list is one month ago.  

https://github.com/crystal-lang/crystal-book/pull/812/files